### DISCUSSION TITLE
Feature Request: Editable User Input

### DISCUSSION THREAD

**Discussion by ford-perfect** [Taewan-P/gpt_mobile#51] [Ideas]
1. Allow copy of userprompt
2. Allow edit of user prompt
3. Make multiple branches accessible
4. Allow to copy complete Chats
5. Multiple system prompts selectable at chat start

**Comment by Taewan-P** [comment]
What do you mean by multiple branches?
Btw we already support copying completed chats

**Comment by Samylol99** [comment]
@Taewan-P 
I'm highly confident that by branching paths, they're referring to when you modify your prompt or regenerate your answer, you can click on the right/left arrows to show the previous/next regenrated reply followed by all the prompts you typed after said reply.
I was also gonna open an issue to request that.
Here are screenshots from ClaudeAI in case it's not clear.
![Screenshot_2024-07-27-04-29-55-672.jpeg](https://github.com/user-attachments/assets/ebdb4ead-8de5-4533-a32b-7eb24837556c)

![Screenshot_2024-07-27-04-30-21-292.jpeg](https://github.com/user-attachments/assets/2cb3fcb9-d38c-4387-8340-fd37c8ac49c4)

**Comment by Taewan-P** [comment]
Aha, I see.

**Comment by Taewan-P** [comment]
Resolved in #119
